web/router.ex

```diff
@@ -17,6 +18,9 @@ defmodule Notox.Router do
     pipe_through :browser # Use the default browser stack
 
     get "/", PageController, :index
+    resources "/users", UserController, only: [:new, :create, :show]
+    resources "/account", AccountController, only: [:show, :edit, :update, :delete]
```

Comment:
Should this be a singleton resource? Read more on that here: https://hexdocs.pm/phoenix/Phoenix.Router.html#resources/4